bids/shell.nix

```diff
@@ -0,0 +1,25 @@
+let
+  pkgs = import <nixpkgs> {};
```

Comment:
Maybe specify the nixpkgs version here as in a year from now this might not work anymore?

Reply:
Good idea, do you have a favorite way to do that?

Reply:
```suggestion
  pkgs = import (builtins.fetchTarball "channel:nixos-unstable-small") {}
```

lgty?

Reply:
:+1: